erehwon/erehwon/settings/__init__.py

```diff
@@ -144,6 +144,8 @@
     os.path.join(BASE_DIR, "../static"),
 ]
 
+# AUTH_USER_MODEL = 'profiles.ErehwonUser'
```

Comment:
this is commented out, it needs to be present before you run django for the "first time' for migrations and the database to be created correctly. You will probably need to drop the db again and remove all migrations.